sections/product-main-template.liquid

```diff
@@ -367,24 +378,34 @@ var waitJquery = function(func) {
     });
   };
 
-      waitJquery(function() {
-        equalizeHeight();
-        enableTabs();
-      });
-
-    {% if product.selected_variant.option1 %}
-      {% assign textInitial = product.selected_variant.option1 %}
-      {% assign colorSelectorInitial = product.selected_variant.option1 | remove: '/' %}
-      {% assign imageSelectorInitial = product.selected_variant.option1 | remove: '/' | remove: ' ' | prepend: 'i' %}
+  window.addEventListener('load', function() {
```

Comment:
what is waitJquery, equalizeHeight and enableTabs?  This really gives me pause as I don't like rearranging the page after it's been rendered as that is a pretty big performance penalty.

Reply:
If we're really having to wait for window.jquery we're doing something wrong. we should be using the document ready

Reply:
equalizeHeight should be done via CSS. let's remove that.

Reply:
enableTabs seems overly complex. Granted, it's been a long time since I've used Jquery for tabs but it should be only a line or two. Try to simplify that.